server/jetstream_cluster.go

```diff
@@ -833,7 +833,11 @@ func (js *jetStream) setupMetaGroup() error {
 	atomic.StoreInt32(&js.clustered, 1)
 	c.registerWithAccount(sacc)
 
-	js.srv.startGoRoutine(js.monitorCluster)
+	js.srv.startGoRoutine(
+		js.monitorCluster,
+		"type", "metaleader",
```

Comment:
Love this idea, but not sure it belongs here. And maybe if it does just supply a single optional map instead?